tests/onnx/test_onnx_v2.py

```diff
@@ -39,6 +46,38 @@ class OnnxUtilsTestCaseV2(TestCase):
     Cover all the utilities involved to export ONNX models
     """
 
+    @classmethod
+    def setUpClass(cls):
+        # Create local checkpoints - one time setup
+        test_model = "mrm8488/bert-tiny-finetuned-squadv2"
+        cls.test_model = test_model
+
+        pt_temp_dir = TemporaryDirectory()
```

Comment:
The `transformers` codebase follows the (standard?) convention of using context managers with `TemporaryDirectory()` to handle temporary files / folders in our test suite. Could you please refactor to match this approach?

Reply:
I originally did this manually because I didn't want files to be written again between unit tests, but it has been refactored in [67416f2](https://github.com/huggingface/transformers/pull/18615/commits/67416f22094bb29b7bdcfe22854ee00ef4e1e112) to match the approach.